lib/plot_test.go

```diff
@@ -20,11 +20,10 @@ func BenchmarkPlotReporter(b *testing.B) {
 			rs[i].Error = "Error"
 		}
 	}
-	rep := NewPlotReporter("Vegeta Plot", &rs)
 	// Start benchmark
 	b.ReportAllocs()
 	b.StartTimer()
 	for i := 0; i < b.N; i++ {
-		rep.Report(ioutil.Discard)
+		NewHTMLPlot("Vegeta Plot", 5000, rs).WriteTo(ioutil.Discard)
```

Comment:
Error return value of `(github.com/tsenart/vegeta/lib.HTMLPlot).WriteTo` is not checked